fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/JobsServlet.java

```diff
@@ -415,11 +433,16 @@ protected void handleResultsResponse(
      * return an Observable wrapping the PreResponse as is.
      *
      * @param preResponse  The PreResponse to be inspected
+     * @param uriInfo  uriInfo object to get uriBuilder
+     * @param paginationParameters  Pagination parameters received through an API
```

Comment:
Missing `request` at the end of this parameter description. Though I would probably rephrase this as something like `the user's requested pagination`